cypress/integration/misc/bpartner_bpartner_relation_spec.js

```diff
@@ -0,0 +1,82 @@
+import { DiscountSchema } from '../../support/utils/discountschema';
+import { BPartner } from '../../support/utils/bpartner';
+import { SalesOrder } from '../../support/utils/sales_order_api';
+
+describe('BPartner relations', function() {
+  const timestamp = new Date().getTime();
+  const customer1Name = `Customer1_${timestamp}`;
+  const customer2Name = `Customer2_${timestamp}`;
+  const salesOrder1Name = `SalesOrder1-${timestamp}`;
+  const bPartnerRelationName = `BPartnerRelation_${timestamp}`;
+  const priceListName = 'Testpreisliste Kunden';
+  const discountSchemaName = `DiscountSchemaTest ${timestamp}`;
+  let relationID = null;
+
+  before(function() {
```

Comment:
Since the tests are generally flaky, i would suggest we don't use the `before` section, just normal `it` surrounded by `describe` maybe. In this way, if something fails we can see it in the cypress video. 

afair `before` is not shown in videos, but i can be wrong.

Reply:
No, this is shown in the log as usual